README.md

```diff
@@ -71,4 +71,4 @@ To add the changes contained in one branch into another branch, or from one fork
 A request for a particular merge to be made. If you have permissions you can manually merge, otherwise you have to request someone else does it. When you create the pull request, anyone who is authorised will be emailed and can review it.  
 * Even if you do have permissions, it is a good idea to create a pull request so a second person can review it before adding to the master.  
 * Try to create a pull request that encompases a set of related changes, if it is two seperate features, creat two pull requests.  
-* If you push more changes to the branch after creating the pull request, they will be automatically included when it is finally merged.  
+* If you push more changes to the branch after creating the pull request, they will be automatically included when it is finally merged.
```

Comment:
You can make a new commit and push to this to the same branch, and it will automatically be included when the pull request is merged